### DISCUSSION TITLE
Having trouble using dagster-airbyte

### DISCUSSION THREAD

**Discussion by alefer559** [dagster-io/dagster#9904] [Q&A] [answered]
Hello! I've been trying to do a simple airbyte sync job on dagster using dagster-airbyte. Everything works up until I hit "Launch Run" where I get the following message:

" Error 1: Post processing at path root:config:host of original value {'env': 'localhost'} failed:
dagster._config.errors.PostProcessingError: You have attempted to fetch the environment variable "localhost" which is not set. In order for this execution to succeed it must be set in this environment. "

And a similar error but with the port {'env': '8000'}

I used the following template on dagster where I input the host, port, and connection id: 

from dagster import job
from dagster_airbyte import airbyte_resource, airbyte_sync_op

my_airbyte_resource = airbyte_resource.configured(
    {
        "host": {"env": "AIRBYTE_HOST"},
        "port": {"env": "AIRBYTE_PORT"},
    }
)

sync_foobar = airbyte_sync_op.configured({"connection_id": "foobar"}, name="sync_foobar")

@job(resource_defs={"airbyte": my_airbyte_resource})
def my_simple_airbyte_job():
    sync_foobar()

I am new to using dagster so if anyone can help it would be greatly appreciated. It appears to be a problem with environmental variables but I'm not sure what to do. Any help is appreciated!

**Comment by sryza** [comment, marked as answer]
I believe that if you replace {"env": "localhost"} with just "localhost" it will get you past this error.
